packages/donor/src/screens/authentication/AuthLoadingScreen.tsx

```diff
@@ -1,11 +1,10 @@
-import Spinner from "../../components/basic/Spinner";
 import Logo from "../logo/Logo";
+import Styles from "./AuthLoadingScreen.module.scss";
 
 export default function AuthLoadingScreen() {
   return (
-    <div style={{ textAlign: "center" }}>
-      <Logo style={{ width: "100%", maxWidth: "320px", height: "400px" }} />
-      <Spinner />
+    <div className={Styles["logo"]}>
+      <Logo size={"large"} />
```

Comment:
Why did you remove the spinner?

Reply:
Unnecessary.
Looks more professional without.

Reply:
@msmeravs Please confirm we want to remove the spinner from the splash screen